crates/cli/src/sync.rs

```diff
@@ -205,6 +211,27 @@ pub async fn config_sync(
                 }
             };
 
+            let token_endpoint_auth_method = match provider.token_endpoint_auth_method {
```

Comment:
maybe this would be better suited as an `.into()` implementation if that's not impossible due to the crates being different

Reply:
The crates where both structures are defined don't depend on each other, which is why we can't implement it